tests/test_interrupted_sync.py

```diff
@@ -183,30 +192,30 @@ def test_run(self):
                 resuming_bookmark_value_utc = self.convert_state_to_utc(resuming_bookmark_value)
                 if stream in new_state['bookmarks'].keys():
                     simulated_bookmark = new_state['bookmarks'][stream]
-                else:
-                    simulated_bookmark = None
-                if stream in new_state['bookmarks'].keys():
                     simulated_bookmark_value = list(simulated_bookmark.values())[0]
+                    expected_resuming_sync_start_time = self.parse_date(first_bookmark_value)
                 else:
+                    simulated_bookmark = None
                     simulated_bookmark_value = None
```

Comment:
I think these can be removed since we hide assertions that use them behind the same if check as above (`if stream in new_state['bookmarks'].keys():`)

Reply:
Removed.